internal/model/measurement.go

```diff
@@ -57,7 +57,7 @@ type Measurement struct {
 
 	// MeasurementStartTimeSaved is the moment in time when we
 	// started the measurement. This is not included into the JSON
-	// and is only used within probe-engine as a "zero" time.
+	// and is only used within ooniprobe-engine as a "zero" time.
```

Comment:
```suggestion
	// and is only used within the ./internal pkg as a "zero" time.
```

Reply:
super minor stuff, but I grepped and there are (still?) a few refs to ooniprobe-engine. I guess at some point they all must go, isn't it?

Reply:
Yes, thanks a lot for this! 🥰